gf-webapp-admin/src/component/filter/Filters.module.scss

```diff
@@ -14,15 +14,138 @@
   gap: 10px;
 }
 
+.filtersRaw {
+  display: flex;
+  justify-content: space-between;
+}
+
+.input {
+  margin: $input-margin;
+  width: $input-width;
+  height: $input-height;
+  padding: $input-padding;
+  font-size: $input-font-size;
+  font-family: $input-font-family;
+  border: $input-border;
+}
+
+.input:focus {
+  outline: none;
+  border: $input-border-focus;
+}
+
+.services {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px;
+  gap: 30px;
+}
+
+.title {
+  font-size: $label-font-size;
+  color: $label-color;
+  margin-right: $label-margin-right;
+}
+
+.label {
+  font-size: $label-font-size;
+  color: $label-color;
+  margin-right: $label-margin-right;
+}
+
+.label span:first-child {
+  display: none;
+}
+
+.labelNone span:first-child {
+  display: none;
+}
+
+label {
+  position: relative;
+  cursor: pointer;
+}
+
+label [type="checkbox"] {
+  display: none;
+}
+
+[type="checkbox"]:checked + span,
+[type="checkbox"]:not(:checked) + span {
+  display: inline-block;
+  position: relative;
+  padding-left: 4px;
+  line-height: 20px;
+  cursor: pointer;
+}
+
+[type="checkbox"]:checked + span:before,
+[type="checkbox"]:not(:checked) + span:before {
+  content: "";
+  margin-left: -24px;
+  // margin-top: -24px;
```

Comment:
Will this commentary be needed in the future?

Reply:
Delete